DRAFT.md

```diff
@@ -1089,31 +1101,22 @@ constexpr indirect& operator=(indirect&& other) noexcept(
     allocator_traits<Allocator>::is_always_equal::value);
 ```
 
-5. _Mandates_: If `allocator_traits<allocator_type>::is_always_equal::value`
+5. _Mandates_: If `allocator_traits<Allocator>::is_always_equal::value`
     is `false`, then `T` is a complete type.
```

Comment:
I think here we require `T` to be a complete type unconditionally, since even in the case where `is_always_equal` is `true`, we might still call the destructor of the existing object in `*this` being replaced by the moved-object taken from `other`.